MobileCenterDistribute/MobileCenterDistribute/Support/MobileCenterDistribute.xcconfig

```diff
@@ -1,3 +1,3 @@
 #include "../../../Global.xcconfig"
 
-OTHER_LDFLAGS=$(inherited) -framework Foundation -framework UIKit -framework CoreTelephony -framework Security
+OTHER_LDFLAGS=$(inherited) -framework Foundation -framework UIKit -framework CoreTelephony
```

Comment:
AFAIK this needs to include SafariServices, too. (It's added in the podspec but not here)